jmx/src/main/java/org/jboss/as/jmx/RemotingConnectorAdd.java

```diff
@@ -47,6 +48,11 @@ private RemotingConnectorAdd() {
     }
 
     @Override
+    protected boolean requiresRuntime(OperationContext context) {
```

Comment:
This is the only oddity in here. Without this, embedded boot consistently fails with: 

19:39:12,063 ERROR [org.jboss.as.controller.management-operation] (Controller Boot Thread) WFLYCTL0013: Operation ("add") failed - address: ([
    ("host" => "master"),
    ("subsystem" => "jmx"),
    ("remoting-connector" => "jmx")
]) - failure description: {
    "WFLYCTL0412: Required services that are not installed:" => [
        "org.wildfly.management.jmx",
        "jboss.remoting.endpoint.management"
    ],
    "WFLYCTL0180: Services with missing/unavailable dependencies" => ["jboss.jmx.remoting-connector-ref is missing [org.wildfly.management.jmx, jboss.remoting.endpoint.management]"]
}